cmd/collectors/commonutils.go

```diff
@@ -166,3 +166,15 @@ func GetDataInterval(param *node.Node, defaultInterval time.Duration) float64 {
 func IsTimestampOlderThanDuration(timestamp float64, duration time.Duration) bool {
 	return time.Since(time.UnixMicro(int64(timestamp))) > duration
 }
+
+// This function would clone the matrix with same instances and metrics, but with different indentifier
+func CloneWithOtherIdentifier(uuid, object string, identifier string, source *matrix.Matrix) *matrix.Matrix {
```

Comment:
object is always == identifier
could simplify and only pass one of them. Works for qtree plugins...

Reply:
not needed.